.github/workflows/CI.yaml

```diff
@@ -69,5 +69,5 @@ jobs:
       - name: Validate openapi spec
         if: ${{ always() }}
         run: |
-          differences=$(diff openapi/openapi.yaml manager/target/generated/openapi.yaml)
-          if [ $? -ne 0 ]; then echo "openapi/openapi.yaml is out of sync. Please move the manager/target/generated/openapi.yaml into openapi/openapi.yaml"; false ; fi
+          differences=$(diff openapi/rhose-api.yaml manager/target/generated/rhose-api.yaml)
```

Comment:
This job runs using the `pull_request_target` event to safely access secrets (aka it takes job definition from main and not from the branch) so the PR check will be red until we merge, as alternative we can do an initial PR that does only the CI change and is backward compatible: like, if `openapi.yaml` exists, use it, otherwise use `rhose-api.yaml`.
It might be an overkill to be honest 🤔 

@r00ta @lucamolteni @sutaakar @manstis 
Any preference?

Reply:
It is fine for me to merge it will failing PR check in such case, the functionality can be tested on merged code.